### PR TITLE
Adding wbgapi library

### PR DESCRIPTION
**Pull request by NejcHirci**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by NejcHirci** [comment]
I allow to be listed as maintainer.

**Comment by NejcHirci** [comment]
@conda-forge/staged-recipes Ready for review.

File: README.md
Comment (BastianZim) [inline README.md:1]: This is the readme of the conda-forge repo. Please do not change anything in here.

File: recipes/wbgapi/meta.yaml
Comment (BastianZim) [inline recipes/wbgapi/meta.yaml:23]: It seems like this is only required when running the package, not when building it.

Ref: https://github.com/tgherzog/wbgapi/blob/a0f8658b7a74ec79256d7b66ff58cb95726e89aa/setup.py#L28

File: recipes/wbgapi/meta.yaml
Comment (BastianZim) [inline recipes/wbgapi/meta.yaml:32]:
```suggestion
    - wbgapi
  commands:
    - pip check
  requires:
    - pip
```

This checks that the dependencies are correctly added.

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: README.md
Comment (NejcHirci) [reply README.md:1]: My bad I unintentionally mistyped.

File: recipes/wbgapi/meta.yaml
Comment (NejcHirci) [reply recipes/wbgapi/meta.yaml:23]: I originally did not include them but the build kept failing, which is why I added them, but will double check.

Comment (BastianZim) [reply recipes/wbgapi/meta.yaml:23]: Hmm, strange. Do you have the logs for that commit?

Comment (NejcHirci) [reply recipes/wbgapi/meta.yaml]: Sadly can't find the azure build log anymore. Will try it without the host requirements again and we ll se if it fails.

Comment (NejcHirci) [reply recipes/wbgapi/meta.yaml]: Okay I think I found the reason for the issue.
`  Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "/home/conda/staged-recipes/build_artifacts/wbgapi_1647278866514/work/setup.py", line 5, in <module>
      from wbgapi.__version__ import __version__ as pkgVersion
    File "/home/conda/staged-recipes/build_artifacts/wbgapi_1647278866514/work/wbgapi/__init__.py", line 9, in <module>
      import requests
  ModuleNotFoundError: No module named 'requests'
  error: subprocess-exited-with-error
`
Don't know why for certain because it is not my repository, but because they import version from wbgapi it requires all the packages imported to wbgapi, so basically all needed for running the library.

Comment (BastianZim) [reply recipes/wbgapi/meta.yaml:23]: Ahh yes, that makes sense – interesting. Ok in that case you'll need to keep it. But I would suggest opening an issue upstream about this because it's a strange setup.

Comment (NejcHirci) [reply recipes/wbgapi/meta.yaml:23]: Will post an issue on the original repo about it. Thank you for your time.

**Comment by BastianZim** [comment]
You still need to remove the changes in the readme.

**Comment by NejcHirci** [comment]
Sorry forgot about that. Here you go.

**Comment by NejcHirci** [comment]
Thank you for the patience its my first time posting on conda-forge.

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
No worries! We're always happy about new contributors. I'll wait for the CI to finish and will then merge it.

**Comment by BastianZim** [comment]
Welcome to conda-forge!

**Pull request merged**
